### DISCUSSION TITLE
How to get a Uri reference to all the selected files in the explorer.

### DISCUSSION THREAD

**Discussion by parithon** [microsoft/vscode-discussions#194] [Extension Development QnA] [answered]
Is it possible to have the explorer/context menu provide an array of the selected files when executing the command? It appears that a reference is only provided to the file from which you opened the context menu. I'm looking to provide a command which executes on multiple files (like the 'Delete' command).

**Comment by worksofliam** [comment]
Don't think it is possible. You can do it with your own treeview, but not others.

https://github.com/microsoft/vscode/issues/3553

**Comment by ArturoDent** [comment, marked as answer]
Is this what you are looking for: https://stackoverflow.com/a/74128313/836330

**Reply by parithon** [reply]
This is precisely what I was looking for, I'm embarrassed I missed that.
